src/js/view/controls/components/chapters.mixin.js

```diff
@@ -2,9 +2,10 @@ import utils from 'utils/helpers';
 import srt from 'parsers/captions/srt';
 
 class Cue {
-    constructor (time, text) {
+    constructor (time, text, playerWidth) {
         this.time = time;
         this.text = text;
+        this.playerWidth = playerWidth;
```

Comment:
@boyntoni you should pass in the model so that in align you can get the latest "containerWidth". That will make it more responsive. Note that the slider width is not the full player width so you might want to do a little more math in there too to match the CSS like the hardcoded `markerWidth`.